src/components/CartCheckout/CartCheckout.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { Context } from '../Context';
+import { Product } from '../../types/Product';
+import './CartCheckout.scss';
+
+export const CartCheckout: React.FC = () => {
+  const { cart } = useContext(Context);
+
+  const [isClicked, setIsClicked] = useState(false);
+
+  const cartTotalPrice = () => {
+    const currentItems = JSON.parse(localStorage.getItem('cart') || '[]');
+
+    return currentItems.reduce(
+      // eslint-disable-next-line max-len
+      (sum: number, current: Product) => sum + (current.price - ((current.price / 100) * current.discount)), 0,
```

Comment:
```suggestion
      (sum: number, current: Product) => 
        sum + (current.price - ((current.price / 100) * current.discount)),
      0,
```
don't disable lint. it's better to format it in another way